app/src/main/java/com/example/mpvcalculatorrx/mvp/view/MvpView.java

```diff
@@ -0,0 +1,4 @@
+package com.example.mpvcalculatorrx.mvp.view;
+
+public class MvpView {
```

Comment:
The same here for interfaces above inheritance

Reply:
Done